app/src/main/java/com/appdev/eateryblueandroid/models/Eatery.kt

```diff
@@ -22,7 +22,9 @@ data class Eatery(
     @Json(name="location") val location: String? = null,
     @Json(name="online_order_url") val onlineOrderUrl: String? = null,
     @Json(name="wait_times") val waitTimes: List<WaitTimeDay>? = null,
-    @Json(name="alerts") val alerts: List<Alert>? = null)
+    @Json(name="alerts") val alerts: List<Alert>? = null,
```

Comment:
I think you can remove the @Json(name="alerts") because the JSON field matches the kotlin one

Reply:
Thats fair - I wanted a way of indicating that a field was a JSON field vs just a field defined in the model (like isFavorite)